### PR TITLE
Wiki Table Editor: Bootstrap Repo with create-react-app

### PR DESCRIPTION
**Pull request by tvillarete**
## Details
Wiki Table Editor was (I believe) our first React project. It was created quite a while ago, and React development has changed a lot since then. Our other repos rely on `create-react-app` to add all the behind-the-scenes React boilerplate necessary to make a modern React application. Our old way of doing things on this repo did not provide a simple way to test changes.
#### Pros
- This repo and all its dependencies are now up-to-date with React v16.4.1 
- We can see the end result simply by running `npm start`
- This fixes a long-standing bug detailed in https://github.com/iFixit/ifixit/issues/17265
#### Cons
- There are absolutely no cons!

## QA
Whoever QA's this come talk to me and I'll work with you to QA this pull.
This pull is a complete overhaul of the `wiki-table-editor` repo, minus the base js logic. 
### Note
When testing this pull outside of the iFixit repo, there will be no styling.

### CODE REVIEW

**Comment by copperwall** [comment]
Is there a reason to commit the `dist` files? When this gets published to NPM the `prepublishOnly` build script generates these and ships them off in the NPM package.

**Comment by tvillarete** [comment]
That's fair, I didn't see that our other repos ignored `/dist`

File: package.json [hunk 1]
```diff
@@ -1,28 +1,33 @@
 {
   "name": "wiki-table-editor",
   "main": "dist/wiki-table-editor.js",
-  "version": "1.1.4",
+  "version": "0.1.0",
+  "private": true,
```
Comment (copperwall) [inline package.json]: I don't think NPM will let us publish this if there's a `private: true` in the package.json. The version should be 1.1.5 to follow semver too.

File: .babelrc [hunk 2]
```diff
@@ -1,7 +1,4 @@
 {
-   presets: ['env', 'react'],
-   plugins: [
-      'transform-object-rest-spread',
-      'transform-import-extension-jsx-to-js'
-   ]
+   presets: ['env', 'react', 'stage-2'],
+   plugins: ['transform-object-rest-spread']
```
Comment (colebemis) [inline .babelrc:3]: Is `transform-object-rest-spread` included in the `stage-2` preset?

File: package.json [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
-    "reactabular-table": "8.6.0",
-    "table-resolver": "3.1.0"
+    "ajv": "^6.5.1",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.1",
+    "react-dnd": "^2.6.0",
+    "react-dnd-html5-backend": "^5.0.1",
+    "react-dnd-multi-backend": "^2.3.8",
+    "react-dom": "^16.4.1",
+    "react-scripts": "1.1.4",
```
Comment (colebemis) [inline package.json:13]: Not a big deal but isn't `react-scripts` a dev dependency?

**Pull request merged**